TableView/FakeDataCell.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  FakeDataCell.swift
+//  TableView
+//
+//  Created by Oleg Yankiwskyi on 4/26/18.
+//  Copyright © 2018 Oleg Yankiwskyi. All rights reserved.
+//
+
+import UIKit
+
+class FakeDataCell: UITableViewCell {
+    
+    static let identifier = "fakeDataCell"
```

Comment:
`extension UITableViewCell {

    class var reuseIdentifier: String {
        return NSStringFromClass(self).components(separatedBy: ".").last!
    }

    static var nib: UINib {
        return UINib(nibName: defaultReuseIdentifier, bundle: nil)
    }
}`
to avoid hardcoding